content/en/docs/tutorials/configuration/kubelet-standalone.md

```diff
@@ -0,0 +1,474 @@
+Visit [releases](https://github.com/containernetworking/plugins/releases) (external link).
+
+{{< tabs name="download_cni_plugins" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+curl -LO https://github.com/containernetworking/plugins/releases/download/v1.5.1/cni-plugins-linux-amd64-v1.5.1.tgz
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+curl -LO https://github.com/containernetworking/plugins/releases/download/v1.5.1/cni-plugins-linux-arm64-v1.5.1.tgz
+{{< /tab >}}
+{{< /tabs >}}
+
+<!--
+```shell
+curl -LO https://github.com/containernetworking/plugins/releases/download/v1.5.1/cni-plugins-linux-amd64-v1.5.1.tgz
+```
+-->
+
+Install:
+
+```shell
+mkdir -p /opt/cni/bin
+```
+
+{{< tabs name="install_cni_plugins" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+tar Cxvf /opt/cni/bin/ cni-plugins-linux-amd64-v1.5.1.tgz
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+tar Cxvf /opt/cni/bin/ cni-plugins-linux-arm64-v1.5.1.tgz
+{{< /tab >}}
+{{< /tabs >}}
+
+<!--
+```shell
+mkdir -p /opt/cni/bin
+tar Cxvf /opt/cni/bin/ cni-plugins-linux-amd64-v1.5.1.tgz
+```
+-->
+
+Quick test:
+
+```shell
+/opt/cni/bin/host-local --version
+```
+
+The output is similar to:
+
+```
+CNI host-local plugin v1.5.1
+CNI protocol versions supported: 0.1.0, 0.2.0, 0.3.0, 0.3.1, 0.4.0, 1.0.0
+```
+
+Configure:
+
+```shell
+mkdir -p /etc/cni
+
+cat << EOF > /etc/cni/cni.template
+{
+  "name": "containerd",
+  "cniVersion": "1.0.0",
+  "plugins": [{
+    "type": "ptp",
+    "ipMasq": true,
+    "ipam": {
+      "type": "host-local",
+      "subnet": "{{.PodCIDR}}",
+      "routes": [
+        {"dst": "0.0.0.0/0"}
+      ]
+    }
+  },{
+    "type": "portmap",
+    "capabilities": {
+      "portMappings": true
+    }
+  }]
+}
+EOF
+```
+
+### Container Runtime
+
+This tutorial suggests downloading the containerd container runtime.
+Visit [releases](https://github.com/containerd/containerd/releases) (external link).
+
+Enable IPv4 packet forwarding:
+
+```shell
+cat <<EOF > /etc/sysctl.d/k8s.conf
+net.ipv4.ip_forward = 1
+EOF
+
+sysctl --system
+```
+The output is similar to:
+
+```
+...
+* Applying /etc/sysctl.d/k8s.conf ...
+net.ipv4.ip_forward = 1
+* Applying /etc/sysctl.conf ...
+```
+
+Download:
+
+{{< tabs name="download_containerd" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+curl -LO https://github.com/containerd/containerd/releases/download/v1.7.22/containerd-1.7.22-linux-amd64.tar.gz
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+curl -LO https://github.com/containerd/containerd/releases/download/v1.7.22/containerd-1.7.22-linux-arm64.tar.gz
+{{< /tab >}}
+{{< /tabs >}}
+
+```shell
+curl -LO https://raw.githubusercontent.com/containerd/containerd/main/containerd.service
+```
+
+<!--
+```shell
+curl -LO https://github.com/containerd/containerd/releases/download/v1.7.22/containerd-1.7.22-linux-amd64.tar.gz
+curl -LO https://raw.githubusercontent.com/containerd/containerd/main/containerd.service 
+```
+-->
+
+Configure:
```

Comment:
Is this true? **Adding text here is optional.**
```suggestion
The CRI support in containerd is already active, but you need to configure a (node local)
pod network (read
[The Kubernetes Network Model](/docs/concepts/services-networking/#the-kubernetes-network-model))
 for more context.

To set up a pod network:
```

Reply:
Not sure about this question. I didn't have to do anything else to make it work beyond what I wrote in the tutorial.